src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -120,3 +121,21 @@ def get_set_cmd_dicts(out: str) -> List[dict]:
             get_cmd='dashboard iscsi-gateway-list',
             get_set_cmd_dicts=get_set_cmd_dicts
         )
+
+    def ok_to_stop(self, daemon_ids: List[str], force: bool = False) -> HandleCommandResult:
+        # if only 1 iscsi, alert user (this is not passable with --force)
+        warn, warn_message = self._enough_daemons_to_stop(self.TYPE, daemon_ids, 'Iscsi', 1, True)
+        if warn:
+            return HandleCommandResult(-errno.EBUSY, '', warn_message)
+
+        # if reached here, there is > 1 nfs daemon. make sure none are down
+        warn_message = ('ALERT: 1 iscsi daemon is already down. ' +
+                        'Please bring it back up before stopping this one')
+        iscsi_daemons = self.mgr.cache.get_daemons_by_type(self.TYPE)
+        for i in iscsi_daemons:
+            if i.status != 1:
```

Comment:
would be great to have an `Enum` here. Maybe like so:

https://github.com/ceph/ceph/blob/39fd806cd45e991deac49c8a938dd6381306facf/src/pybind/mgr/orchestrator/module.py#L33-L37

do you want to add this to the `_interface.py`?